hedera-node/hedera-app/src/test/resources/bootstrap.properties

```diff
@@ -79,8 +79,8 @@ contracts.keys.legacyActivations=1058134by[1062784]
 contracts.knownBlockHash=
 contracts.localCall.estRetBytes=32
 contracts.maxGasPerSec=15000000
-contracts.maxKvPairs.aggregate=500_000_000
-contracts.maxKvPairs.individual=163_840
+contracts.maxKvPairs.aggregate=500000000
```

Comment:
why remove the underscores in this value? They make it easier to read and verify that the correct value is set

Reply:
The underscores were causing some of the HAPI tests to fail in the modularized code with `NumberFormatException`. Apparently, the logic for parsing the config values with the underscores has not been implemented in some cases (most likely the new config system)? We probably went overboard on removing them but we wanted to be sure that the tests would pass.

Reply:
> Can you file an issue for Hendrik to add underscore support to number properties in the new config system? I agree, without underscores, we're likely to make a mistake.

Will do.

Reply:
an issue was already created by @david-bakin-sl 
https://github.com/hashgraph/hedera-services/issues/9589

Reply:
Under scores restored and ticket opened